src/Compiler/Driver/CompilerOptions.fs

```diff
@@ -1333,6 +1333,13 @@ let advancedFlagsFsc tcConfigB =
             Some(FSComp.SR.optsEmitDebugInfoInQuotations ())
         )
 
+        CompilerOption(
```

Comment:
true = always strict
false = always non-strict
Not specified = follow based on current LangVersion 

?

Reply:
Yes, that's correct.